script/Ciel/Day 6/06_01_CIEL06_1_1.txt

```diff
@@ -132,6 +131,7 @@ I sit down on the sofa, holding back the pain running through my whole body.
 [sha:84255c3b15e88d7018db00d48a81719184862361]{
 -- Page 46, Offset 13269. Mods: @k, @e.
 -- 「卑怯でしょう。そんな顔をされたら気がゆる―――
+// this one is ゆる while arc is in kanji 緩, should they just be the same in en?
```

Comment:
same in en. I talked about this with Ross the other day but for _some reason_ there are tons of little minor differences in the source text. Sometimes the sentence is structured slightly differently. I couldn't tell you why. It's like they were being written from memory or copied by hand or something.

Anyways, just consolidate with Arc if the line is obviously the same.